.github/workflows/preDeploy.yml

```
@@ -38,9 +38,7 @@ jobs:
   skipDeploy:
     runs-on: ubuntu-latest
     needs: chooseDeployActions
-    if: |
-      ${{ needs.chooseDeployActions.outputs.isStagingDeployLocked == 'true' }} &&
-      ${{ needs.chooseDeployActions.outputs.isVersionBumpPR == 'false' }}
+    if: ${{ needs.chooseDeployActions.outputs.isStagingDeployLocked == 'true' }} && ${{ needs.chooseDeployActions.outputs.isVersionBumpPR == 'false' }}
```

I think we need this to be like this:

```suggestion
    if: ${{ needs.chooseDeployActions.outputs.isStagingDeployLocked == 'true' && needs.chooseDeployActions.outputs.isVersionBumpPR == 'false' }}
```